components/automate-ui/src/app/pages/compliance-landing/compliance-landing.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component } from '@angular/core';
+import { RoutePerms } from 'app/components/landing/landing.component';
+
+@Component({
+  selector: 'app-compliance-landing',
+  template: '<app-landing [routePerms]="routeList"></app-landing>',
+  styleUrls: []
+})
+export class ComplianceLandingComponent {
+
+  // order determined by admin-sidebar template and is vetted by admin-sidebar unit tests
```

Comment:
s/admin-sidebar/compliance-reporting-sidebar/ in the comment above...
and then make it true by adding a unit test to compliance-reporting-sidebar.component.spec.ts
that mirrors the test "has paths consistent with settings-landing" in admin-sidebar.component.spec.ts

Extra credit: Add a couple lines of comments to the underlying landing.component.ts file that explains how to use it, now that you have gone through the exercise. (I should have done that when I created it. 😁 )